debian/ceph-common.postinst

```diff
@@ -62,6 +62,8 @@ case "$1" in
                -d $SERVER_HOME   \
                -g $SERVER_GROUP  \
                $SERVER_USER
+       # Unlock $SERVER_USER in case it is locked from an uninstall
+       usermod -e 99999 -U $SERVER_USER
```

Comment:
@tchaikov Could you please explain why this just checks for the existence of the file `/etc/shadow` and not the existence of the locked user ?